bindings/CXX11/adios2/cxx11/fstream/ADIOS2fstream.tcc

```diff
@@ -15,6 +15,17 @@
 
 #include "adios2/core/Stream.h"
 
+#if defined(_MSC_VER) && !defined(__clang__)
```

Comment:
@chuckatkins some of the files have those pragma guards which disables a whole warning in the file. The reason of doing this instead of solving the warning is that either solving the warning added another problem or that for the sake of code clarity, getting rid of the warning obscured the code.